packages/scripts/src/commands/app-data/convert/report/report.utils.ts

```diff
@@ -0,0 +1,34 @@
+//  TODO - move to generic location (possibly object-utils once #2423 merged)
```

Comment:
#2423 should be ready to merge now

Reply:
Thanks, refactored with 8e4b098311ff4678de1141f15ae36191ba2b70e6